src/ui/components/RecommendedBadge/styles.scss

```diff
@@ -0,0 +1,24 @@
+@import '~ui/css/styles';
+
+.RecommendedBadge {
+  align-items: center;
+  color: #e25920;
```

Comment:
We should probably use a variable for this. This specific color does not seem to exist in `photon-colors`, so we'll need to create our own. I wasn't sure if I should just create one in this file, or in a higher level `vars` file.